.gitignore

```diff
@@ -1 +1,3 @@
 target/
+.idea
```

Comment:
If `zinc/src/test/resources/bin` is added, then it's because it has generated jars/classesdirs. But I don't see any of those jars in git history. If they are presented, they should also be removed. Otherwise, I don't see why we should special case this directory, it hasn't generated anything for me at least.